### DISCUSSION TITLE
Can't pay for the invoice

### DISCUSSION THREAD

**Discussion by agsmartwolf** [swellstores/community#223] [Help] [answered]
The modal window about payment is appearing constantly, I've updated payment info but there is an error:

<img width="1239" alt="Screenshot 2023-04-18 at 21 01 54" src="https://user-images.githubusercontent.com/109105178/232851107-ca5d505e-b5f6-4a75-b2e2-09e1532d03a2.png">

**Comment by ericingram** [comment]
🤦 I guess we should put a floor on invoices at $0.50

**Comment by steven-k1** [comment, marked as answer]
@agsmartwolf We applied a credit for the $0.34, and the invoice was processed. You should be all set now.
